add-on/jar.mn

```diff
@@ -49,10 +49,10 @@
   content/shared/vendor/classnames.js (chrome/content/shared/vendor/classnames.js)
   content/shared/vendor/lodash.js     (chrome/content/shared/vendor/lodash.js)
 #ifdef DEBUG
-  content/shared/vendor/react.js           (content/shared/vendor/react.js)
-  content/shared/vendor/react-dom.js       (content/shared/vendor/react-dom.js)
+  content/shared/vendor/react.js           (chrome/content/shared/vendor/react.js)
+  content/shared/vendor/react-dom.js       (chrome/content/shared/vendor/react-dom.js)
 #else
-  content/shared/vendor/react.js           (content/shared/vendor/react-prod.js)
-  content/shared/vendor/react-dom-prod.js  (content/shared/vendor/react-dom-prod.js)
+  content/shared/vendor/react.js           (chrome/content/shared/vendor/react-prod.js)
+  content/shared/vendor/react-dom-prod.js  (chrome/content/shared/vendor/react-dom-prod.js)
```

Comment:
Actually, this seems to be wrong in multiple ways. We want the file to be called `react-dom.js` always so `<script>` references work without `#ifdef`s?